database/redis/database.go

```diff
@@ -79,7 +79,14 @@ func NewDatabase(logger moira.Logger, config Config, source DBSource) *DbConnect
 
 // NewTestDatabase use it only for tests
 func NewTestDatabase(logger moira.Logger) *DbConnector {
-	return NewDatabase(logger, Config{Addrs: []string{"0.0.0.0:6379"}}, testSource)
+	return NewDatabase(logger, Config{
+		Addrs: []string{"0.0.0.0:6379"},
+	}, testSource)
+}
```

Comment:
I see that some tests rely on existing dbSource in test daatbase object, would it make more sense to add it as argument to `NewTestDatabase` func with `default=testSource`?

Reply:
There are not default arguments in Go )